### PR TITLE
Online event docs

### PR DESCRIPTION
**Pull request by corneelwille**
Entry API
* attendance mode (add / update / delete)
* online url (add / update / delete)

Search API
* filter on attendance mode

--
Ticket: https://jira.uitdatabank.be/browse/III-4742

### CODE REVIEW

File: content/json-ld-crud-api/latest/events/event-attendance-mode.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+---
+
+# Attendance mode
+
+Indicates if an event is happening online, offline, or both (mixed):
+
+* `offline`: the event takes places on a real location
```
Comment (LucWollants) [inline content/json-ld-crud-api/latest/events/event-attendance-mode.md:8]:
```suggestion
* `offline`: the event takes places on a physical location
```

File: content/json-ld-crud-api/latest/events/event-attendance-mode.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* `online`: the events takes places on a virtual location
```
Comment (LucWollants) [inline content/json-ld-crud-api/latest/events/event-attendance-mode.md:9]:
```suggestion
* `online`: the events takes places on a nil location
```

File: content/json-ld-crud-api/latest/events/event-attendance-mode.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* `mixed`: the event takes places both on a real location and a virtual location
```
Comment (LucWollants) [inline content/json-ld-crud-api/latest/events/event-attendance-mode.md:10]:
```suggestion
* `mixed`: the event takes places both on a physical location and a nil location
```

File: content/json-ld/latest/events/event-attendance-mode.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| eventId	| uuid | unique identifier for an event | 15277b21-f91d-42d3-8f14-a4ff66fe538a |
+
+## Request body
+
+| Property	| Type | Description |
+|--|--|--|
+| attandanceMode | string | attendance mode of the event (online, offline or mixed) |
```
Comment (LucWollants) [inline content/json-ld/latest/events/event-attendance-mode.md:42]: I would also add `location` with a remark it is optional

File: content/json-ld/latest/events/event-attendance-mode.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+## Response
+
+* `204 No Content` : request successful
+* `400 Bad Request` : incorrect method, payload or URI
+* `401 Unauthorized` : expired JWT or the user behind the JWT does not have permission to perform this request
```
Comment (LucWollants) [inline content/json-ld/latest/events/event-attendance-mode.md:48]: There are more possible errors, see https://publiq.stoplight.io/docs/uitdatabank/3f46181d4b607-attendance-mode-update

But maybe they were never documented before?

File: content/json-ld/latest/events/event-online-url.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| eventId	| uuid | unique identifier for an event | 15277b21-f91d-42d3-8f14-a4ff66fe538a |
+
+## Request body
+
+| Property	| Type | Description |
+|--|--|--|
+| onlineUrl | url | url where the online event takes place (e.g. the link of the livestream) |
```
Comment (LucWollants) [inline content/json-ld/latest/events/event-online-url.md:38]: Using "takes place" feels like a physical location. Not sure how to phrase it properly. Maybe something like "The url where the online event can be watched/witnessed/followed"

Comment (LucWollants) [reply content/json-ld/latest/events/event-online-url.md:38]: I noticed you used the term "participate" inside the SAPI3 documentation, maybe that is a good term?

File: content/search_api_3/latest/searching/offers/attendance-mode.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+|--|--|
+| `offline` | visitors can attend the event at a physical location | 
+| `online` | visitor can only participate online | 
+| `mixed` | it is possible to attend the event both on a physical location or online | 
+
+You can filter by attendance mode using two methods:
+* URL parameter
+* Advanced queries
+
+Note that places will always have the attendance mode "offline", only events can have a different attendance mode.
```
Comment (LucWollants) [inline content/search_api_3/latest/searching/offers/attendance-mode.md:18]: I am not sure a place even has this property. The code is written only for events.

File: content/search_api_3/latest/searching/offers/attendance-mode.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+Using the `attendanceMode` URL parameter, you can limit the results you get back to events with a certain attendance mode.
+Example to get results that have the attendanceMode "online":
+
+```
+GET https://search.uitdatabank.be/offers/?attendanceMode=online
+```
+
+## Advanced queries
+
+You can also search by status using [advanced queries](https://documentatie.uitdatabank.be/content/search_api_3/latest/reference/advanced-queries.md),
```
Comment (LucWollants) [inline content/search_api_3/latest/searching/offers/attendance-mode.md:31]:
```suggestion
You can also search by attendance mode using [advanced queries](https://documentatie.uitdatabank.be/content/search_api_3/latest/reference/advanced-queries.md), 
```

File: content/json-ld/latest/events/event-attendance-mode.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| eventId	| uuid | unique identifier for an event | 15277b21-f91d-42d3-8f14-a4ff66fe538a |
+
+## Request body
+
+| Property	| Required? |  Type | Description |
+|--|--|--|--|
+| attandanceMode | true | string | attendance mode of the event (online, offline or mixed) |
+| location | false | URI | URI of the (real) location |
```
Comment (LucWollants) [inline content/json-ld/latest/events/event-attendance-mode.md:43]: I think it's better to stay consistent and use `physical` here

File: content/json-ld/latest/events/event-attendance-mode.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
+---
+---
+
+# Attendance mode
+
+Indicates if an event is happening online, offline, or both (mixed):
+
+* `offline`: the event takes places on a real location
```
[the remark this answers is not in the document]
Comment (corneelwille) [reply content/json-ld/latest/events/event-attendance-mode.md:8]: Thanks, I corrected it a bit too hurried I think.

[the remark this answers is not in the document]
Comment (LucWollants) [reply content/json-ld/latest/events/event-attendance-mode.md:8]: No worries 😉

**Pull request merged**